obsplus/utils/stations.py

```diff
@@ -38,15 +49,42 @@ def df_to_inventory(df) -> obspy.Inventory:
     df
         A dataframe which must have the same columns as the once produced by
         :func:`obsplus.stations_to_df`.
+    client
```

Comment:
This needs to be filled in...
Perhaps "a client that can be used to download station response information"?